src/abstractdataframe/show.jl

```diff
@@ -573,61 +573,193 @@ function _show(io::IO,
                df::AbstractDataFrame;
                allrows::Bool = !get(io, :limit, false),
                allcols::Bool = !get(io, :limit, false),
-               splitcols = get(io, :limit, false),
                rowlabel::Symbol = :Row,
                summary::Bool = true,
                eltypes::Bool = true,
-               rowid=nothing,
-               truncstring::Int)
+               rowid = nothing,
+               truncate::Int = 32,
+               kwargs...)
+
     _check_consistency(df)
 
-    # we will pass around this buffer to avoid its reallocation in ourstrwidth
-    buffer = IOBuffer(Vector{UInt8}(undef, 80), read=true, write=true)
+    aux       = names(df)
+    names_len = textwidth.(aux)
+    maxwidth  = max.(9, names_len)
+    names_mat = permutedims(aux)
+    types     = eltype.(eachcol(df))
```

Comment:
this is a style we normally use
```suggestion
    aux = names(df)
    names_len = textwidth.(aux)
    maxwidth = max.(9, names_len)
    names_mat = permutedims(aux)
    types = eltype.(eachcol(df))
```